tests/Behat/Content/Node/ProjectContentContext.php

```diff
@@ -97,4 +112,50 @@ public function alterProjectFields(BeforeParseEntityFieldsScope $scope): void {
     ]);
   }
 
+  /**
+   * Creates media Remote video with the specified URLs.
+   *
+   * Usage example:
+   *
+   * Given the following remote videos:
+   *   | url   |
+   *   | url 1 |
+   *   |  ...  |
+   *
+   * @Given the following Remote video(s):
```

Comment:
Let's do it as in "@Given the following image(s):" - "@Given the following remote video(s):"

Reply:
done